common/history/buildings/07_south_america.txt

```diff
@@ -1122,6 +1122,13 @@
 				reserves=1
 				activate_production_methods={ "pm_basic_port" }
 			}
+			
+			create_building={
+				building="building_gold_mine"
```

Comment:
If you want to add gold mine, ensure the state has capped_resources gold.

[20:26:26][state.cpp:4070]: State Côte du Café has 2 of building type Gold Mines and can only support 0, its level will be reduced.